torch/csrc/jit/passes/quantization/helper.cpp

```diff
@@ -14,7 +14,7 @@ struct FuncArg {
 using AtenFuncArgs = std::vector<FuncArg>;
 using CallFuncArgs = std::vector<FuncArg>;
 
-// White lists for quantizable operators
+// Lists of allowed quantizable operators
```

Comment:
can you change this to supported? Thanks!

Reply:
Done.

Reply:
Ping.